### ISSUE TITLE
Push notification info text is misleading

### ISSUE THREAD

**Issue by armhub** [tutao/tutanota#1145]
It says you have to remove an entry if you do not want to receive push notifications any more, but actually you have to deactivate it then. We should clarify what the difference between deleting and deactivating is.

**Comment by ganthern** [comment]
current:
```
List of all recipients receiving notifications for the current Tutanota user. You can delete an entry if you do not wish to receive notifications anymore.
```

my proposal: 
```
List of all recipients receiving notifications for this user. You can deactivate entries if you do not wish to receive notifications or delete them for devices you don't use anymore.
```

**Comment by armhub** [comment]
Sounds good!

**Comment by vitoreiji** [comment]
Solved by 4c52f6d99eff7bbbc8b3d5135708b7306b9de448
